examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -47,6 +47,8 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
     var defaultNonSamplingDayRange = 1500;
     var defaultSamplingDayRange = 1;
     var defaultSamplingSize = 10;
+    var defaultPointmapSamplingDayRange = 30;
+    var defaultPointmapLimit = 500*1000;
```

Comment:
please estimate how many results can be put in the websocket by the given hard-coded 5M limit.